lib/pbio/drv/legodev/legodev_pup.c

```diff
@@ -431,7 +447,15 @@ void pbdrv_legodev_init(void) {
         pbio_dcmotor_get_dcmotor(legodev, &dcmotor);
         legodev->ext_dev->uart_dev = pbdrv_legodev_pup_uart_configure(legodev_data->ioport_index, port_data->uart_driver_index, dcmotor);
 
+        // legodev driver is started after all other drivers, so we
+        // assume that we do not need to wait for this to be ready.
```

Comment:
I would add an assert statement here so that we could at least catch this in debug builds to catch future changes that might break this assumption.